File: Julia/Algorithms/linear_search.jl
function linear_search(li, item)
    idx = 1

    while idx <= length(li)
        if li[idx] == item
            return idx
        end
        idx += 1
    end
    return -1
end

li = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

println(linear_search(li, -1))

File: JavaScript/Algorithms/linearSearch.js
const linearSearch = (array, item) => {
    let idx = 0;

    while (idx < array.length) {
        if (array[idx] === item) {
            return idx;
        }
        idx++;
    }
    return -1;
}

let arr = [1, 2, 3, 4, 5];

console.log(linearSearch(arr, 4));